foxy/distribution.yaml

```
@@ -1420,8 +1420,8 @@ repositories:
   fastrtps:
     doc:
       type: git
-      url: https://github.com/eProsima/Fast-RTPS.git
-      version: v2.0.2
+      url: https://github.com/eProsima/Fast-DDS.git
+      version: v2.1.3
```

```suggestion
      version: 2.1.x
```

(this will make it match the source entry below)